armory/docker/management.py

```diff
@@ -20,7 +20,7 @@ def __init__(
         self,
         image_name,
         runtime: str = "runc",
-        envs: dict = None,
+        envs: dict = {},
```

Comment:
Having a dict as a default parameter, since it is mutable (and hence can have weird side-effect problems), and is generally not recommended.

Instead, I recommend that you keep it None in the method definition and then add this below:
```
if envs is None:
    envs = {}
```